API-strategie-extensies/ext-geospatial.md

```diff
@@ -123,18 +123,48 @@ The guiding priciples for CRS support:
 - Exchange format (notation) RD and Web Mercator X Y in meters: `195427.5200 311611.8400`
 
 <div class="rule" id="api-40">
-  <p class="rulelab"><strong>API-40</strong>: Pass the coordinate reference system (CRS) of the request and the response in the headers</p>
-  <p>The coordinate reference system (CRS) for both the request and the response are passed as part of the request headers and reponse headers. In case this header is missing, send the HTTP status code <code>412 Precondition Failed</code>.</p>
+  <p class="rulelab"><strong>API-40</strong>: Pass the coordinate reference system (CRS) of the request and the desired CRS of the response as parameters</p>
+  <p>The coordinate reference system (CRS) for both the request and the response are passed as parameters. The CRS of the geometry in the response is specified using a header.</p>
 </div>
 
-The following headers are purely meant for negotiation between the client and the server. Depending on the application, the request not only contains geometries but also specific meta data, e.g. the original realistion including the collection date.
+This method conforms to the standard OGC API Features part 2: Coordinate Reference Systems by Reference. 
+
+Use the following parameters: 
+- A parameter `crs` to specify the CRS used in a request. The value of the parameter is a URI identifier of the CRS. If the `crs` parameter is specified, then the coordinates of all geometry-valued properties in the response document must be presented in the requested CRS. If the requested CRS is not available, respond with HTTP status code 400.
```

Comment:
Welke parameter kan een client meegeven om aan te geven in welk CRS de geometrie in de response moet zijn?

Er zijn verschillende scenario's mogelijk:
1. geen geometrie in request - geen geometrie in response -> geen CRS negotiation nodig
2. geen geometrie in request - geometrie in response -> een client kan aangeven in welk CRS hij de geometrie in de response wil hebben, maar met welke parameter? De server geeft in de content-crs header aan in welk CRS de geometrie in de response is.
3. geometrie in request - geen geometrie in response -> een client geeft met de crs parameter aan in welk CRS de geometrie in de request is
4. geometrie in request - geometrie in response -> een client geeft aan in welk CRS de geometrie in de request is en in welk CRS hij de geometrie in de response wil ontvangen, de server geeft in de response aan in welk CRS de geometrie in de response is.
5. Als de client een CRS opgeeft waarin hij de geometrie in de response wil ontvangen, maar de server ondersteund deze niet, geeft de server in de (fout) response dan aan welke CRS-en hij wel ondersteund? Welke header wordt daar dan voor gebruikt?

Reply:
> Als de client een CRS opgeeft waarin hij de geometrie in de response wil ontvangen, maar de server ondersteund deze niet, geeft de server in de (fout) response dan aan welke CRS-en hij wel ondersteund? Welke header wordt daar dan voor gebruikt?

Uitzoekpunt hoe dit in OGC API Features part 2 zit.

Reply:
http://docs.opengeospatial.org/is/18-058/18-058.html#_parameter_crs  geeft in requirement 11 A het antwoord:
![image](https://user-images.githubusercontent.com/80040145/158609382-e46dbe6d-a725-4748-bcc0-92aa5d112eae.png)